go.mod

```diff
@@ -443,3 +443,5 @@ replace (
 	k8s.io/pod-security-admission => k8s.io/kubernetes/staging/src/k8s.io/pod-security-admission v0.0.0-20221208195143-b46a3f887ca9
 	k8s.io/sample-apiserver => k8s.io/kubernetes/staging/src/k8s.io/sample-apiserver v0.0.0-20221208195143-b46a3f887ca9
 )
+
+replace github.com/containers/storage => github.com/kwilczynski/storage v1.13.2-0.20241211191335-e3c1975dedc2
```

Comment:
is this really meant to point to an unreleased version?

Reply:
See: 

- https://github.com/cri-o/cri-o/pull/8838#discussion_r1882354878